benchmark/Benchmark.py

```diff
@@ -8,40 +8,51 @@
 """
 
 
-import pandas
+try:
+	import pandas
+except ImportError as ex:
+	pandas = None
 import sys
 import warnings
 import math
 import os
 import numpy
-import matplotlib.pyplot as plt
-import seaborn
+try:
+	import matplotlib.pyplot as plt
+except ImportError as ex:
+	plt = None
+try:
+	import seaborn
+except ImportError as ex:
+	seaborn = None
 from time import gmtime, strftime
 import signal
 
 
 import networkit
 
 from util import *
+from networkit.support import MissingDependencyError
 import nk
 
 try:
 	import networkx
 	import nx
 except ImportError as ex:
-	print("networkx not available")
+	networkx = None
```

Comment:
What about the `nx` variable here? We should either consistently define all module symbols as `None` if the packages are not available, or consistently only set one symbol per package to `None`.

Reply:
An alternative would be to have a Boolean `have_networkx` variable that we set to `False` if the import failed. Maybe that is the cleanest solution.